qiskit/pulse/schedule/experiment_condition.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2019, IBM.
+#
+# This source code is licensed under the Apache License, Version 2.0 found in
+# the LICENSE.txt file in the root directory of this source tree.
+
+"""
+Configurations for pulse experiments.
+"""
+from typing import Dict
+
+from qiskit.pulse.channels import OutputChannel
+from qiskit.pulse.exceptions import PulseError
+
+
+class UserLoDict:
```

Comment:
If this is required, should it be a more general `ExpConfig` or something along these lines?